ndb/tests/conftest.py

```diff
@@ -40,3 +43,28 @@ def reset_state():
     yield
     model.Property._FIND_METHODS_CACHE.clear()
     model.Model._kind_map.clear()
+
+
+@pytest.fixture
+def environ():
+    """Copy of ``os.environ``"""
+    original = os.environ.copy()
```

Comment:
This copy isn't needed since you overwrite `os.environ`?

Reply:
raise FingersFasterThanBrainError()

Fixed: 78f02e7